frontend/packages/metal3-plugin/src/status/host-status.ts

```diff
@@ -1,5 +1,5 @@
 import { K8sResourceKind, MachineKind, NodeKind } from '@console/internal/module/k8s';
-import { getHostOperationalStatus, getHostProvisioningState } from '../selectors';
+import { getHostOperationalStatus, getHostProvisioningState, getHostErrorType } from '../selectors';
 import {
```

Comment:
HOST_STATUS_REGISTRATION_ERROR, HOST_STATUS_REGISTERING, HOST_STATUS_PROVISIONING and HOST_STATUS_PROVISIONING_ERROR are unused now, so they should be removed.